src/content/docs/infrastructure/infrastructure-ui-pages/hosts-new-view.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Infrastructure Hosts page (new view)
+tags:
+  - Infrastructure
+  - Infrastructure monitoring UI
+  - Infrastructure UI
+  - Hosts page
+metaDescription: Use the Hosts page to view the most important metrics from across your server/host environment.
+---
+
+import infrastructureHostsNewSummary from 'images/infrastructure_screenshot-full_hosts-new-summary.png'
+
+import infrastructureSavedViewsBttn from 'images/infrastructure_icon_saved-views-bttn.png'
+
+import infrastructureExportCSV from 'images/infrastructure_icon_export-csv.png'
+
+import infrastructureHostNewSystem from 'images/infrastructure_screenshot-full_host-new-system.png'
+
+import infrastructureHostNewNetwork from 'images/infrastructure_screenshot-full_host-new-network.png'
+
+import infrastructureHostNewProcesses from 'images/infrastructure_screenshot-full_host-new-processes.png'
+
+import infrastructureHostNewStorage from 'images/infrastructure_screenshot-full_host-new-storage.png'
+
+import infrastructureHostNewContainers from 'images/infrastructure_screenshot-full_host-new-containers.png'
+
+
+<Callout title="PREVIEW FEATURE">
```

Comment:
@x8a I think the NerdLife post on release types specifies that releases in the "PUBLIC PREVIEW" stage should only use "PREVIEW" for external readers. This should probably be spelled out in our style guide, too.

Reply:
You mean I should remove `FEATURE`?

Reply:
@x8a, yes, I think the word "PREVIEW" should be removed, but you can use the word "feature" in the body of the callout (This is a preview of the feature...." That NerdLife post was just trying to bring some consistency to the label.